content/2025/_index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Home
+menu:
+    main:
+        weight: 1
+        identifier: home-2025
+year: 2025
+type: home
+layout: home
+---
+
+<!-- JUMBO -->
+
+{{% jumbo img="/images/2025/background-2025.jpg" imgLabel="QCrypt 2025 background" logo="/images/2025/QCrypt_2025_logo.png" %}}
```

Comment:
it should be background_2025.jpg, that's why it is not displayed on the main page!